src/styles/blocks/container.scss

```
@@ -0,0 +1,27 @@
+.container {
+  --column-count: 1;
+
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    .container {
+      --column-count: 2;
+    }
```

A selector is unnecessary here because nesting is used
```suggestion
    --column-count: 2;
```